.eslintrc.js

```diff
@@ -15,6 +15,19 @@ module.exports = require('@sumup/foundry/eslint')(
       'emotion/jsx-import': 'error',
     },
     overrides: [
+      {
+        files: ['**/*.js'],
+        rules: {
+          '@typescript-eslint/no-explicit-any': 'off',
+          '@typescript-eslint/no-unsafe-call': 'off',
+          '@typescript-eslint/no-unsafe-return': 'off',
+          '@typescript-eslint/no-unsafe-assignment': 'off',
+          '@typescript-eslint/no-unsafe-member-access': 'off',
+          '@typescript-eslint/restrict-template-expressions': 'off',
+          '@typescript-eslint/restrict-plus-operands': 'off',
+          '@typescript-eslint/explicit-module-boundary-types': 'off',
```

Comment:
These rules are suddenly applied to JavaScript files. Need to look into this further and fix it in Foundry before the stable v4 release.